src/scss/login.scss

```diff
@@ -0,0 +1,151 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;600;700&family=Source+Sans+Pro:wght@200;300;400;600;700&display=swap');
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Inter', sans-serif;
+    font-family: 'Source Sans Pro', sans-serif;
+}
```

Comment:
Not happy to see this. This can overwright all default margin and paddings that elements may set explicitly. It is also very slow, as it will apply itself to litterally all elements and trace down their familiar path. The use of the * selector in this way is bad practice, as it can make things difficult to debug, it is much slower, and breaks inheritance entirely. If you'd like things to be this way by default these chould be applied to the body tag. This should also be applied to any element that breaks inheritance seperately.

Reply:
To elaborate, if we move the files load order these things can also be overwitten or may overwrite things that it currently doesn't

Reply:
remove all / everything selector